dotCMS/src/main/java/com/dotcms/rest/UserResource.java

```diff
@@ -45,7 +44,7 @@
 	@Path("/getloggedinuser/{params:.*}")
 	@Produces("application/json")
 	@Deprecated
-	public Response getLoggedInUser(@Context HttpServletRequest request, @PathParam("params") String params) throws DotDataException, DotSecurityException,
+	public Response getLoggedInUser(@Context HttpServletRequest request, @PathParam("params") String params) throws DotDataException,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)